Formula/s/spoofdpi.rb

```diff
@@ -1,8 +1,8 @@
 class Spoofdpi < Formula
   desc "Simple and fast anti-censorship tool written in Go"
   homepage "https://github.com/xvzc/SpoofDPI"
-  url "https://github.com/xvzc/SpoofDPI/archive/refs/tags/0.10.12.tar.gz"
-  sha256 "1ef78cc8afaf59216619cfd17b01dcfc554524fe2ee5a8365f239ba3eef63b70"
+  url "https://github.com/xvzc/SpoofDPI/archive/refs/tags/v0.10.12.tar.gz"
+  sha256 "c3b4d903a8a210f539727bfb7afa075d4e132a2cd8402481fa55f7a0606217a2"
```

Comment:
The version has not changed and the previous `0.10.12` tag [seems to have been removed](https://github.com/xvzc/SpoofDPI/tags). Can you please confirm what's going on?

Reply:
Absence of the 'v' prefix in version numbers breaks `go install` with the tag `@latest`. Go claims to use semantic versioning, but unfortunately it doesn't. To overcome this issue, all SemVer tags were prefixed with `v`.

Reply:
Thanks for confirming, @Ledorub.